doc/notes/189_reporting_ant.adoc

```diff
@@ -0,0 +1,224 @@
+= Protocol Verifier Reporting
+
+xtUML Project Analysis Note
+
+== 1 Abstract
+
+This note provides analysis of reporting requirements for the Munin
+Protocol Verifier.  These requirements will flow backwards into the
+strategy for logging and reporting.
+
+== 2 Introduction and Background
+
+Early in Project Munin, the engineering development team erred toward
+over-logging and over-reporting, because clear requirements for both were
+emerging.  The understanding of the application has grown.  Requirements
+may now be surmised with some hope of usefulness.
+
+== 3 Requirements
+
+=== 3.1 Dashboard Requirements
+
+A dashboard shall be constructed which puts key metrics on display.
```

Comment:
Is the dashboard something that the MUNIN project team will be constructing, or is the client responsible for developing this albeit consuming reported output from the PV?